code/datums/components/tackle.dm

```diff
@@ -369,7 +369,7 @@
 
 	if(HAS_TRAIT(target, TRAIT_CLUMSY))
 		defense_mod -= 2
-	if(HAS_TRAIT(target, TRAIT_FAT)) // chonkers are harder to knock over
+	if(HAS_TRAIT(target, TRAIT_FAT) || HAS_TRAIT(target, TRAIT_OFF_BALANCE_TACKLER)) // chonkers are harder to knock over
```

Comment:
Alright now you would remove the `HAS_TRAIT(target, TRAIT_FAT)` and instead, when we `ADD_TRAIT(mob, TRAIT_FAT, OBESITY` you would have `ADD_TRAIT(mob, TRAIT_OFF_BALANCE_TACKLER, OBESITY)`